packages/components/input-number/src/input-number.vue

```diff
@@ -60,7 +60,14 @@
       @focus="handleFocus"
       @input="handleInput"
       @change="handleInputChange"
-    />
+    >
+      <template #prefix>
```

Comment:
```suggestion
      <template #prefix v-if="$slots.prefix || prefixText">
```
A judgment should be added to render only when the corresponding attribute or slot exists.

Reply:
It may be necessary to add it here to prevent unnecessary label content from appearing when no slot is added.